### PR TITLE
Fix perf tests report when there are no tests

### PR DESCRIPTION
**Pull request by tavplubix**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

See https://github.com/ClickHouse/ClickHouse/pull/56797

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 55987297a145a30c1ff9a07b861a2a9a06a4191b with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/56881/55987297a145a30c1ff9a07b861a2a9a06a4191b/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6895723484">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/56881/55987297a145a30c1ff9a07b861a2a9a06a4191b/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: docker/test/performance-comparison/compare.sh [hunk 1]
```diff
@@ -226,6 +229,13 @@ function run_tests
         test_files=("${test_files[@]}")
     fi
 
+    if [ "$run_only_changed_tests" -ne 0 ]; then
+        if [ ${#test_files[@]} -eq 0 ]; then
+            time "$script_dir/report.py" > report.html
```
Comment (antonio2368) [inline docker/test/performance-comparison/compare.sh:234]: did you plan to call it with `--no-tests-run`?

Comment (tavplubix) [inline docker/test/performance-comparison/compare.sh:234]:
```suggestion
            time "$script_dir/report.py" --no-tests-run > report.html
```

**Review by antonio2368 — APPROVED** [review]

**Comment by tavplubix** [comment]
Integration tests (asan) [1/4] - https://github.com/ClickHouse/ClickHouse/issues/56627
Stateless tests (debug, s3 storage) [4/6] - `00993_system_parts_race_condition_drop_zookeeper` - looks like a known issue with removal of blobs of empty parts, I will fix it

**Pull request merged**